staging/src/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -173,6 +175,7 @@ func NewCmdGet(parent string, f cmdutil.Factory, streams genericclioptions.IOStr
 
 	cmd.Flags().StringVar(&o.Raw, "raw", o.Raw, "Raw URI to request from the server.  Uses the transport specified by the kubeconfig file.")
 	cmd.Flags().BoolVarP(&o.Watch, "watch", "w", o.Watch, "After listing/getting the requested object, watch for changes. Uninitialized objects are excluded if no object name is provided.")
+	cmd.Flags().BoolVarP(&o.IncludeDeleteEvents, "include-delete-events", "", false, "Events are delte every 60m. This will include delete events on watch")
```

Comment:
```suggestion
	cmd.Flags().BoolVarP(&o.IncludeDeleteEvents, "include-delete-events", "", false, "Events are deleted every 60m. This will include delete events on watch")
```